src/HomeworkLoopsArrays/ConvertNumberToArray.java

```diff
@@ -0,0 +1,32 @@
+package HomeworkLoopsArrays;
+
+import java.sql.SQLOutput;
+import java.util.Scanner;
+
+/**
+ * Created by Anahit Khachatryan  on  18, Sep, 2020
+ */
+public class ConvertNumberToArray {
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+        //1. enter number
+        System.out.println("Enter number: ");
+        int num = scanner.nextInt();
+        int[] digits = Integer.toString(num).chars().map(c -> c - '0').toArray();
```

Comment:
es uzum ei while-ov u 10-i vra bajanelov aneir)
public int[] convertDigitsToArray(int n) {

    int [] temp = new int[String.valueOf(n).length()]; //calculate the length of digits
    int i = String.valueOf(n).length()-1 ;  //initialize the value to the last index
    do {
        temp[i]= n%10;
        n = n/10;
        i--;
    }while(n>0);
    return temp;
}

motavor senc mi ban..